com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JdtSourceLookUpProvider.java

```diff
@@ -211,7 +211,7 @@ public JavaBreakpointLocation[] getBreakpointLocations(String sourceUri, SourceB
                 // In future, we could consider supporting to update the breakpoint to a valid
                 // location.
                 BreakpointLocationLocator locator = new BreakpointLocationLocator(astUnit,
-                        sourceLine, true, true);
+                        sourceLine, true, true, astUnit.getPosition(sourceLine, 0), 0);
```

Comment:
This is a limitation of the upstream ValidBreakpointLocationLocator. It requires the selection offset to search for line breakpoints on nested chained or multiline lambdas.